src/main/java/uk/gov/pay/connector/gateway/GatewayClient.java

```diff
@@ -163,6 +186,14 @@ private void incrementFailureCounter(MetricRegistry metricRegistry, String metri
         metricRegistry.counter(metricsPrefix + ".failures").inc();
     }
 
+    private void incrementPrometheusFailureCounter(PaymentGatewayName gatewayName, String gatewayAccountType, OrderRequestType orderRequestType) {
+        this.gatewayOperationsFailures.labels(
```

Comment:
`this` refers to the current object. As `gatewayOperationsFailures` is a static (class) variable, we can remove reference to this.

```suggestion
       gatewayOperationsFailures.labels(
```

Reply:
Thanks, removed 👍